OSDOCS-15381 created oidc assembly and updated topic map (#97396)
openshift/openshift-docs: modules/zero-trust-manager-oidc-config.adoc

  jwtIssuer: <jwt_issuer_domain> #<3>
----
<1> The trust domain to be used for the SPIFFE identifiers.
<2> The name of the SPIRE Agent UNIX socket.

> <3> The JSON Web Token (JWT) issuer domain. The default value is set to the value specified in `oidc-discovery.$trustDomain`.

Reviewer: ```suggestion
<3> The JSON Web Token (JWT) issuer domain. The value must be a valid URL.
```
